client/js/main.js

```diff
@@ -5,14 +5,22 @@ import store from './store/store';
 import router from './router';
 Vue.config.productionTip = false;
 
-const app = new Vue({
-  vuetify,
-  store,
-  router,
-  render: (h) => h(App),
-}).$mount('#app');
+async function initApp() {
+  // Make sure we try and reestablish a user's session if one exists
+  // We do this before the app mounts to preempt the reauth/logout logic
+  await store.dispatch('requestStatus');
```

Comment:
This is how we make sure a user stays logged in on refresh. We need to verify with the sails server first and update the store before we mount the Vue app.